### ISSUE TITLE
[rps] Channel Provider should be loaded by webpack

### ISSUE THREAD

**Issue by lalexgap** [statechannels/monorepo#709]
Currently we're referencing `channelProvider` to ensure `window.channelProvider` is defined:
```
 async enable() {
    // TODO: This is really ugly!!
    // TODO: Use webpack to import channelProvider
    // tslint:disable-next-line: no-unused-expression
    channelProvider;
```

We should use webpack or update channel-provider to be a simple library we can import.

**Comment by snario** [comment]
Can the parent frame (i.e., `rps`) retrieve it from the `iframe` like this?

```js
var channelProvider = document.getElementById("iframeid").contentWindow.channelProvider;
```

If so that could work. In the MetaMask extension scenario we would expect the extension to inject `window.channelProvider` onto the page.

**Comment by geoknee** [comment]
I think this was solved in #745.
